tests/playwright/sanity/modules/nested-accordion/nested-accordion.test.js

```diff
@@ -163,26 +240,23 @@ test.describe( 'Nested Accordion @nested-accordion', () => {
 
 			const nestedAccordionWidgetId = '48f02ad',
 				frame = editor.getPreviewFrame(),
-				nestedAccordion = frame.locator( '.e-n-accordion' ).filter( { hasText: 'One' } ),
+				nestedAccordion = frame.locator( '.e-n-accordion' ),
 				nestedAccordionTitle = frame.locator( 'summary' ).first().filter( { hasText: 'One' } );
 
-			await test.step( 'Widget Screenshot matches intended design', async () => {
-				expect( await frame.locator( '.e-n-accordion' ).first().screenshot( { type: 'jpeg', quality: 90 } ) ).toMatchSnapshot( 'nested-carousel-title-and-icons.jpg' );
-			} );
```

Comment:
This test has been deleted as it includes an error in it.
The accordion title default value is set to `h2` and not `div` as it should be.
as well, screenshots tests should be added once the widget is completely built.

Reply:
@davseve good save, default was changed to H2 on ED-10516. I've extracted this screenshot as a regression screenshot on the above PR to catch rendering issues in editor and in the front. I agree if there are big structural changes we can skip this test. I used to to catch a rendering issues caused by html escaping. I've seen this happen on 2 tasks, so this test is important to catch such instances. 2nd break happened inside the NA container element.

Reply:
Thanks :)
This test indeed is really important but it contains an error.
The accordion's title default tag should be `div` and not `h2`
@avivu  

I think we should add this error in the accordion last dev task.